packages/components/src/menu/tests/chromatic/Menu.chroma.jsx

```diff
@@ -0,0 +1,747 @@
+import { Avatar } from "@components/avatar";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong